package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "bufferutil": "^4.0.1",
     "canvas": "^2.6.1",
+    "ch-logging": "git+ssh://git@github.com/companieshouse/ch-structured-logging-node.git#a3e0dcc57f91f63803e02ceef2cc5e4038de9a19",
```

Comment:
Presume this repo will be properly released via Concourse with a release version at some point?

Reply:
I guess so, it's not our repo